frontend.conf

```diff
@@ -33,4 +46,106 @@ server {
     }
 }
 
+# -----------------------------------------------------------------------------#
+#                                                                              #
+# 2. Advanced Example: Landing page, login/logout button to handle OIDC kflow  #
```

Comment:
I think it's better to have one example with commented out directives than to have multiple examples which can lead to additional complexity or potential security risks. More details below.

Reply:
Great comment and idea to have one example!